dnf/cli/cli.py

```diff
@@ -837,8 +837,9 @@ def _parse_commands(self):
         command_cls = self.cli_commands.get(basecmd)
         if command_cls is None:
             logger.critical(_('No such command: %s. Please use %s --help'),
-                                  basecmd, sys.argv[0])
-            logger.info(_("It could be a DNF plugin command."))
+                            basecmd, sys.argv[0])
+            logger.info(_("It could be a DNF plugin command, "
+                          "try: \"dnf install 'dnf-command(%s)'\""), basecmd)
```

Comment:
installing plugins via provides works correctly only with "/usr/bin/dnf". I don't want to mess these provides with py2/3 versions. Using `dnf-2/3` is on users own risk - it doesn't have to fit with plugin path.